### PR TITLE
LPS-69449 & LPS-69599

### PR DESCRIPTION
**Pull request by blzaugg**

### CODE REVIEW

File: portal-web/docroot/html/taglib/aui/select/end.jsp [hunk 1]
```diff
@@ -42,14 +42,16 @@
...
 
-	select.on(
-		'keydown',
-		function(event) {
-			if (event.which == 27) {
-				event.stopPropagation();
-			}
-		}
-	);
+	<c:if test="<%= BrowserSnifferUtil.isIe(request) || BrowserSnifferUtil.isEdge(request) %>">
```
Comment (blzaugg) [inline portal-web/docroot/html/taglib/aui/select/end.jsp]: Not sure of your original conditional. Please test.

Comment (blzaugg) [inline portal-web/docroot/html/taglib/aui/select/end.jsp]: Not sure of your original conditional. Please test.

**Comment by jonmak08** [comment]
ci:reopen

**Comment by jonmak08** [comment]
Hey @blzaugg,

I was just talking to @robert-frampton and we are going to try to push this through to upstream again.  Do you mind reviewing this and fixing the rebase errors and resending me a new pull request?

Please let me know if you have any questions. Thanks!

**Comment by blzaugg** [comment]
Will look at this tomorrow.

**Pull request closed without merging**